build.gradle.kts

```diff
@@ -55,19 +59,26 @@ plugins {
 
 spinePublishing {
     projectsToPublish.addAll(
+        ":mc",
+        ":tool-base",
+        ":plugin-base",
+        ":plugin-testlib"
     )
     targetRepositories.addAll(
         PublishingRepos.cloudRepo,
         PublishingRepos.cloudArtifactRegistry
     )
+    // Skip the `spine-` part of the artifact name to avoid collisions with the currently "live"
+    // versions.
+    spinePrefix.set(false)
```

Comment:
Are we going to change this in the future? If so, let's create an issue right away and reference it here. At least we'll that there's smth we have missed/not done yet easier.